terraform.tf

```diff
@@ -4,6 +4,9 @@ terraform {
       source  = "hashicorp/azurerm"
       version = ">= 3.7.0"
     }
+    azapi = {
+      source = "azure/azapi"
```

Comment:
I would add a version constraint for major releases. Probably not necessary, but it's good practice.

Reply:
added version constraint